mapie/classification.py

```diff
@@ -1158,35 +631,10 @@ def fit(
                 self.y_pred_proba_raps, self.y_raps
             )
 
-        # Conformity scores
-        if self.method == "naive":
-            self.conformity_scores_ = (
-                np.empty(y_pred_proba.shape, dtype="float")
-            )
-        elif self.method in ["score", "lac"]:
-            self.conformity_scores_ = np.take_along_axis(
-                1 - y_pred_proba, y_enc.reshape(-1, 1), axis=1
-            )
-        elif self.method in ["cumulated_score", "aps", "raps"]:
-            self.conformity_scores_, self.cutoff = (
-                self._get_true_label_cumsum_proba(y, y_pred_proba)
-            )
-            y_proba_true = np.take_along_axis(
-                y_pred_proba, y_enc.reshape(-1, 1), axis=1
-            )
-            random_state = check_random_state(self.random_state)
-            u = random_state.uniform(size=len(y_pred_proba)).reshape(-1, 1)
-            self.conformity_scores_ -= u * y_proba_true
-        elif self.method == "top_k":
-            # Here we reorder the labels by decreasing probability
-            # and get the position of each label from decreasing
-            # probability
-            self.conformity_scores_ = get_true_label_position(
-                y_pred_proba, y_enc
-            )
-        else:
-            raise ValueError(
-                "Invalid method. " f"Allowed values are {self.valid_methods_}."
+        # Compute the conformity scores
```

Comment:
why don't we put those if conditions directly into the conformity scores ? Then all the code will be simplified as their won't be any need to have kwargs for the `predict_set` method

Reply:
Done 👍 I hope the proposal suits you